ts/Stock/RangeSelector/RangeSelector.ts

```diff
@@ -2069,9 +2223,20 @@ class RangeSelector {
     ): void {
         const chart = this.chart;
         merge(true, chart.options.rangeSelector, options);
+        merge(true, this.options, options);
```

Comment:
Do we need to merge it twice? Could something like this be enough?
```js
this.options = chart.options.rangeSelector = merge(chart.options.rangeSelector, options);
```

I assume those are references to the same object anyway, according to `init()` method